dashboard/app/views/devise/unlocks/new.html.haml

```diff
@@ -7,4 +6,4 @@
     %br/
     = f.email_field :email, :autofocus => true
   %div= f.submit "Resend unlock instructions"
-= render "devise/shared/old_links"
+= render "devise/shared/new_links"
```

Comment:
i had a TODO for myself to just delete this devise route since we don't use it (from as far as i can tell). nobody seemed to have strong preferences (i asked in slack a while back), so i'm leaving for now but am removing the `_old_links` partial to that view can be deleted.

however, i'd still like to remove this view if we aren't using it. let me know what you all think!

Reply:
![drag-to-trash](https://i.imgur.com/v1zXSTi.gif?1)

Reply:
Looks like we're still using it here?  https://github.com/code-dot-org/code-dot-org/blob/8990f16a29d8a1cf9dba0ffb220bd424e8542810/dashboard/app/views/devise/passwords/edit.html.haml#L14